docs/source/reference-observations.rst

```diff
@@ -0,0 +1,144 @@
+
+Data Files
+----------
+* `sfa_reference_sites.csv`
+   The master list of reference sites. See the comment at the top of this file
+   for descriptions of its fields. The file contains extra fields that are not
+   found in the Solar Forecast Arbiter API schema for Sites. These fields are
+   for use with the source network's API and are stored in the
+   `extra_parameters` field when the site is created for use in subsequent
+   updates.
+
+* `<network>_reference_sites.json`
+   Network specific files containing site and observation metadata in the API's
+   JSON format. These are used when the master CSV does not contain all of
+   the columns needed to accurately define a site or observation.
+
+Modules
+-------
+* :py:mod:`solarforecastarbiter.io.reference_observations.reference_data`
+   This module coordinates the initialization and update process. It also 
+   contains the `NETWORKHANDLER_MAP` dictionary, which maps network names to
+   the correct Network Handler. The functions in the module are utilized by the
+   CLI `referencedata` command.
+
+* :py:mod:`solarforecastarbiter.io.reference_observations.common` 
+   A module containing network-agnostic utility functions. For things like
+   posting data to the Solar Forecast Arbiter API and filtering reference data
+   by network.
+
+Network Handlers
+****************
+Network Handlers are network specific modules that implement a handful of
+functions such that they share a common interface. See
+:py:mod:`solarforecastarbiter.io.reference_observations.surfrad` for an
+example.
+
+Required network handler functions:
+
+* `initialize_site_observations(api, site)`
+   Create an observation at the site for each variable available from the
+   network.
+    * api: :py:class:`solarforecastarbiter.io.api.APISession`
+    * site: :py:class:`solarforecastarbiter.datamodel.Site`
+
+
+* `initialize_site_forecasts(api, site)`
+   Create a forecast fore each observation at the site.
+    * api: :py:class:`solarforecastarbiter.io.api.APISession`
+    * site: :py:class:`solarforecastarbiter.datamodel.Site`
+
+
+* `update_observation_data(api, sites, observations, start, end)`
+   Retrieve data from the network then format and post it to each observation
+   at the site.
+    * api: :py:class:`solarforecastarbiter.io.api.APISession`
+    * sites: :py:class:`solarforecastarbiter.datamodel.Site`
+    * sites: :py:class:`solarforecastarbiter.datamodel.Site`
+    * start: datetime
+    * end: datetime
+
+
+* (optional) `adjust_site_parameters(site)`
+   In instances where the master site CSV does not contain enough metadata about
+   the site, (e.g. when a PV plant requires `modeling_parameters`) this function
+   may be used to update the site metadata before it is posted to the API.
+    * site: dict
+
+
+Available Network Handlers
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* NOAA (The National Oceanic and Atmospheric Administration)
+   * SURFRAD: Surface Radiation Budget Network
```

Comment:
For consistency with DOE and NREL, don't group together the NOAA networks. I don't have a good understanding of the history of any of them, but I'm guessing they might as well be at different institutions anyways, so a flat list seems appropriate from that perspective too.